app/helpers/avatar_helper.rb

```diff
@@ -0,0 +1,13 @@
+module AvatarHelper
+  def avatar
+    if !current_user.avatar.url.nil?
+      image_tag current_user.avatar,
+        size: "150", 
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Trailing whitespace detected.